pkg/flare/archive_test.go

```diff
@@ -247,6 +249,35 @@ func TestCleanDirectoryName(t *testing.T) {
 	assert.True(t, !directoryNameFilter.MatchString(cleanedHostname))
 }
 
+func TestZipLogFiles(t *testing.T) {
+	srcDir, err := ioutil.TempDir("", "logs")
+	require.NoError(t, err)
+	// defer os.RemoveAll(srcDir)
+	dstDir, err := ioutil.TempDir("", "TestZipLogFiles")
+	require.NoError(t, err)
+	// defer os.RemoveAll(dstDir)
```

Comment:
```suggestion

	dstDir, err := ioutil.TempDir("", "TestZipLogFiles")
	require.NoError(t, err)
```

Reply:
Good catch, I commented these out to troubleshoot but these are actually meant to be uncommented.

Fixed it, could you have another look?